### ISSUE TITLE
Exponential back-off times should have a configuration option

### ISSUE THREAD

**Issue by jirevwe** [frain-dev/convoy#662]
There's no configuration option for exponential back-off times. [See here](https://github.com/frain-dev/convoy/pull/634/files#diff-8c908455a10ca93136c49e598f3195b7793276df7b4ed5eeebcef98f6256617aR15-R26)

**Comment by Subomi** [comment]
@jirevwe Thoughts - Should we hold off on this pending the `subscriptions` collection? Since this will be changing alongside that. cc @danvixent

**Comment by jirevwe** [comment]
Hmm, I think it can be refactored in after it's merged.

**Comment by Subomi** [comment]
@oneEyedSunday No, it won't. But I think that API is simpler than this one. In fact, I'd prefer us to merge it in one go. That is 
1. Derive the `retryInterval`
2. Expose a single configuration property - `IntervalSeconds`
